docker/nginx/conf.d/include/proxy-cache-downloads

```diff
@@ -11,4 +11,11 @@ proxy_cache_bypass $cookie_nocache $arg_nocache;
 # no cache - this will ignore cache on request (status MISS)
 # and does not store file in cache under no condition
 set_if_empty $nocache "0";
-proxy_no_cache $nocache;
+
+# map skyd env variable value to "1" for true and "0" for false (expected by proxy_no_cache)
+set_by_lua_block $skyd_disk_cache_enabled {
+    return os.getenv("SKYD_DISK_CACHE_ENABLED") == "true" and "1" or "0"
```

Comment:
Wow, this `== "true" and "1" or "0"` is so confusing at a first glance to a Lua newbie like myself 😅 

Thanks for that comment above, it helped.